packages/http-client-csharp/generator/Microsoft.Generator.CSharp/src/Models/Types/ModelTypeProvider.cs

```diff
@@ -13,9 +13,20 @@ namespace Microsoft.Generator.CSharp
     public sealed class ModelTypeProvider : TypeProvider
     {
         private readonly InputModelType _inputModel;
+        private CSharpMethod? _publicConstructor;
+        private CSharpMethod? _serializationConstructor;
+        private CSharpMethod? _emptyConstructor;
```

Comment:
Why do we need the empty ctor?

Reply:
It is generated in the autorest.csharp repo for `Unbranded-TypeSpec`project [here](https://github.com/Azure/autorest.csharp/blob/feature/v3/test/UnbrandedProjects/Unbranded-TypeSpec/src/Generated/Models/RoundTripModel.cs#L133-L135), so I used the same logic to generate the same output.

Reply:
It is generated in the autorest.csharp repo for `Unbranded-TypeSpec`project [here](https://github.com/Azure/autorest.csharp/blob/feature/v3/test/UnbrandedProjects/Unbranded-TypeSpec/src/Generated/Models/RoundTripModel.cs#L133-L135), so I used the same logic to generate the same output. Should I remove this method generation in MGC?

Reply:
MRW requires an empty ctor